docker/docker-compose.yml

```diff
@@ -210,15 +210,15 @@ services:
       - mydefault
 
   rdf-fingerprinter-fuseki:
-    container_name: fuseki-fingerprinter
+    container_name: rdf-fingerprinter-fuseki
     image: meaningfy/fuseki:latest
     env_file:
       - .env
     environment:
       - ADMIN_PASSWORD=${RDF_FINGERPRINTER_FUSEKI_ADMIN_PASSWORD}
       - JVM_ARGS=${RDF_FINGERPRINTER_FUSEKI_JVM_ARGS}
     ports:
-      - ${RDF_FINGERPRINTER_FUSEKI_PORT}:3030
+      - 3020:3030
```

Comment:
In the final solution, this should not be hardcoded, a variable shall be here instead.